website/snippets/_enterprise-permissions-table.md

```diff
@@ -84,7 +84,7 @@ The project roles enable you to work within the projects in various capacities.
 | Metadata                 |   R   |    R    |       R        |     R     |     R     |     R     |      R      |     R    |                |     R       |     R      |        |
 | Permissions              |   W   |         |       R        |     R     |     R     |           |             |          |                |             |     W      |        |
 | Profile                  |   W   |    R    |       W        |     R     |     R     |     R     |             |          |                |     R       |     R      |        |
-| Projects                 |   W   |    W    |       W        |     W     |     W     |     R     |      R      |          |                |     R       |     W      |        |
+| Projects                 |   W   |    R    |       R        |     R     |     R     |     R     |      R      |          |                |     R       |     R      |        |
```

Comment:
in dbt-cloud code, what I see is `Admin`, `Database Admin`, `Git Admin`, and `Team Admin` all have Project write permission